app/uk/gov/hmrc/cipphonenumberverification/controllers/InternalAuthAccess.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cipphonenumberverification.controllers
+
+import uk.gov.hmrc.internalauth.client.{IAAction, Predicate, Resource, ResourceLocation, ResourceType}
+
+object InternalAuthAccess {
```

Comment:
can we make this a trait and then mixin?

Reply:
Why?

Reply:
It's what we are doing elsewhere, including your other PR. I believe using a mixin to 'mix in' behaviour is more appropriate than creating a singleton object